File: Chatmvp - Java/app/src/main/java/nosql/anew/chatmvp/views/ILoginView.java
package nosql.anew.chatmvp.views;

/**
 * Created by saksham on 20/6/17.
 */

public interface ILoginView {
    void showToast(String message);
    void authSuccessful();
    void showRoomDialog();
    void changeButtonText();
    void startChatActivity(String roomName);
}

File: Chatmvp - Kotlin/app/src/main/java/nosql/anew/kotlinmvp/views/ILoginView.kt
package nosql.anew.kotlinmvp.views

/**
 * Created by Sam on 27/04/18.
 */
interface ILoginView {
    fun showToast(message: String)
    fun authSuccessful()
    fun showRoomDialog()
    fun changeButtonText()
    fun startChatActivity(roomName: String)
}
